config.default.yaml

```diff
@@ -408,18 +412,66 @@ solving:
     clip_p_max_pu: 0.01
     skip_iterations: true
     track_iterations: false
-    #nhours: 10
+    # nhours: 10
+
   solver:
     name: gurobi
-    threads: 4
-    method: 2 # barrier (=ipm)
-    crossover: 0
-    BarConvTol: 1.e-5
-    FeasibilityTol: 1.e-6
-    AggFill: 0
-    PreDual: 0
-    GURO_PAR_BARDENSETHRESH: 200
-
+    options: gurobi-default
+
+  solver_options:
+    highs-default:
+      # refer to https://ergo-code.github.io/HiGHS/dev/options/definitions/
+      threads: 4
+      solver: "ipm"
+      run_crossover: "off"
+      small_matrix_value: 1e-6
+      large_matrix_value: 1e9
+      primal_feasibility_tolerance: 1e-5
+      dual_feasibility_tolerance: 1e-5
+      ipm_optimality_tolerance: 1e-4
+      parallel: "on"
+      random_seed: 123
+    gurobi-default:
+      threads: 4
```

Comment:
@FabianHofmann has it been a specific reason for adding these updates to the PR? 

The linopy-integration PR #796 introduces the same changes, so we'll have them in the workflow in any case. Do you see any advantages for testing from having these changes earlier? Just trying to understand the logic :)

Reply:
Yes, these are needed to enable the incremental config changes (see above). But this will be easy to merge with the linopy PR